### ISSUE TITLE
Include improvements by bookmark package

### ISSUE THREAD

**Issue by septatrix** [latex3/hyperref#144]
I recently found a reference to the `bookmark` package inside of the `hyperref` manual. One of the major advantages is that it only needs one latex run to produce the desired output.

Are there any plans to add these changes in the normal hyperref package?

**Comment by u-fischer** [comment]
Yes. At the end there should be only one method to create the outlines.

**Comment by septatrix** [comment]
Is this already in the works or just some future project?

**Comment by u-fischer** [comment]
It is in the works. But the time must be counted in months, not weeks or days.

**Comment by septatrix** [comment]
Okay thanks for the information - I did not knew how vast the scope of this is
